### ISSUE TITLE
New uCoin instances have a forkWindow of zero

### ISSUE THREAD

**Issue by c-geek** [ucoin-io/ucoin#258]
We often see this these days:

```js
{
  "ucoin": {
    "software": "ucoind",
    "version": "0.12.4",
    "forkWindowSize": 0
  }
}
```

My guess is it comes from `sync` command that put the value to zero temporarily, but seems to persist it also.

**Comment by M5oul** [comment]
I don't think it's fixed:
* http://metab.ucoin.io/node/summary
* http://88.174.120.187:45001/node/summary
* http://urodelus.ddns.net:9201/node/summary

**Comment by c-geek** [comment]
That's bad news :-/ thanks for the report.

**Comment by c-geek** [comment]
Fixed by b6b228896edc746f2d17e4c588a9de56fe1f7d29.
